retrofit2-clients/src/main/java/com/palantir/remoting3/retrofit2/Retrofit2Client.java

```diff
@@ -56,7 +71,19 @@ private Retrofit2Client() {}
     public static <T> T create(
             Class<T> serviceClass,
             UserAgent userAgent,
-            Refreshable<ClientConfiguration> config) {
+            Refreshable<ClientConfiguration> config,
```

Comment:
Just one more request: could we keep the `Refreshable<ClientConfiguration>` as the last argument?  (It's just to match the pattern from functional pattern where currying is a thing.)